### ISSUE TITLE
Textures are loaded incorrectly (color space)

### ISSUE THREAD

**Issue by FeuerTiger86** [maikramer/Materialize#32]
I noticed that when loading textures like normal maps, they aredarker than when they were first created, which results in a very different full material preview. I loaded the project in Unity and determined that this is apparently because the project is set to linear color space but the texture files are sRGB (at least I believe that this is the case, I'm no expert on color spaces). Changing the color space to sRGB in the Project Settings seems to correct this behavior.

**Comment by maikramer** [comment]
Yes i know, after updating Unity i got some issues, but i'm not developping this anymore. The most stable version for real, is 0.3.7, [Here](https://github.com/maikramer/Materialize/releases/tag/v0.3.7), the later versions i did without proper testing. Thanks.

**Comment by nscTechArt** [comment]
> I noticed that when loading textures like normal maps, they aredarker than when they were first created, which results in a very different full material preview. I loaded the project in Unity and determined that this is apparently because the project is set to linear color space but the texture files are sRGB (at least I believe that this is the case, I'm no expert on color spaces). Changing the color space to sRGB in the Project Settings seems to correct this behavior.

bro, you just helped me out, THANK U!!!
